src/plot_profile/plot_rs/plot_rs.py

```diff
@@ -142,15 +144,21 @@ def get_axis_limits(df, params, user_inputs_dict):
             temp_min = user_inputs_dict["temp_min"]
         if user_inputs_dict["temp_max"]:
             temp_max = user_inputs_dict["temp_max"]
-        return windvel_min, windvel_max, temp_min, temp_max
+        if user_inputs_dict["pot_temp_min"]:
+            pot_temp_min = user_inputs_dict["pot_temp_min"]
+        if user_inputs_dict["pot_temp_max"]:
+            pot_temp_max = user_inputs_dict["pot_temp_max"]
+        return windvel_min, windvel_max, temp_min, temp_max, pot_temp_min, pot_temp_max
 
     # case 2: use standard axis limits
     # > temperature range:   -100 - 30 [°C]
     # > windvelocity range:     0 - 30 [m/s]
+    # > potential temperature range:    275 - 325 [K]
     if user_inputs_dict["standard_settings"]:
         windvel_min, windvel_max = 0, 30
         temp_min, temp_max = -100, 30
-        return windvel_min, windvel_max, temp_min, temp_max
+        pot_temp_min, pot_temp_max = 275,325
```

Comment:
I suggest to align the units of temperature and potential temperature: either both in °C or both in K

Reply:
(or then at least remove pot_temp from the list of variables which get plotted by default)

Reply:
we removed pot_temp from the default variables, but we left the units of temperature in °C and that from potential temperature in K